client/src/components/RatingsAndReviews/ReviewsList.jsx

```diff
@@ -1,11 +1,24 @@
 import React from 'react';
 import Review from './Review.jsx';
+import axios from 'axios';
 
 const ReviewsList = (props) => {
+  const reviewHelpful = (reviewId) => {
+    axios.put('/reviews/:review_id/helpful', { review_id: reviewId })
+      .then(info => console.log('info:', info))
+      .catch(err => err);
+  };
+
+  const reportReview = (reviewId) => {
+    axios.put('/reviews/:review_id/report', { review_id: reviewId })
+      .then(info => console.log('info:', info))
```

Comment:
same here

Reply:
I'm trying to maintain consistency with the API endpoints to avoid confusion.